dnf-plugins-extras.spec

```diff
@@ -32,6 +32,7 @@ Extras Plugins for DNF.
 
 %package -n %{name}-common-data
 Summary:        Common data files for Extras Plugins for DNF
+Provides:       %{name} = %{version}-%{release}
```

Comment:
You mean revert this commit: https://github.com/rpm-software-management/dnf-plugins-extras/commit/2ae5079678c3711b50cea54d4fc5848e8ba0618e